src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-otel-applications-kubernetes.mdx

```diff
@@ -0,0 +1,252 @@
+---
+title: Link OpenTelemetry-instrumented applications to Kubernetes
+tags:
+  - Integrations
+  - Kubernetes integration
+  - OpenTelemetry
+  - Link apps and services
+metaDescription: "Connect New Relic's Kubernetes monitoring with applications monitored with OpenTelemetry."
+---
+
+import k8sOtelValidation from 'images/k8s-otel-validation.png'
+
+If you’re a developer running apps in Kubernetes, you can use New Relic to understand how Kubernetes infrastructure affects your OpenTelemetry-instrumented applications.
+
+After completing the steps below, you can use the New Relic UI to correlate application-level metrics from OpenTelemetry with Kubernetes infrastructure metrics. This will help you view the entire landscape of your telemetry data and work across teams to get faster mean time to resolution (MTTR) for issues in your Kubernetes environment.
+
+## Overview: How we correlate the data [#correlate]
+
+The steps in this guide enable your application to inject infrastructure-specific metadata into the telemetry data. The result is that the New Relic UI is populated with actionable information. Here are the steps you'll take to get this going:
+
+  * In each application container, define an environment variable to send telemetry data to the collector
+  * Deploy the OpenTelemetry collector as a `DaemonSet` in [agent mode](https://opentelemetry.io/docs/collector/getting-started/#agent) with `resourcedetection`, `resource`, `batch` and `k8sattributes` processors to inject relevant metadata (cluster, deployment, and namespace names)
+
+## Prerequisites [#prereqs]
+
+To be successful with the steps below, you should already be familiar with OpenTelemetry and Kubernetes and have done the following:
+
+  * Installed New Relic’s [Kubernetes integration](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure) in your cluster
+  * Instrumented your applications with [OpenTelemetry](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup/), and successfully sent data to New Relic via OpenTelemetry Protocol (OTLP)
+
+If you have general questions about using collectors with New Relic, see our [Introduction to OpenTelemetry Collector with New Relic](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-intro).
+## Configure your application to send telemetry data to the OpenTelemetry collector [#instrument]
+
+To set this up, you need to add a custom snippet to the `env` stanza of your Kubernetes YAML file. We have an example below that shows the snippet for a sample front-end microservice (`Frontend.yaml`). The snippet includes two sections that do the following:
+
+  * **Section 1:** Ensure that the telemetry data is sent to the collector. This sets the environment variable OTEL_EXPORTER_OTLP_ENDPOINT with the host IP. It does this by calling the downward API to pull the host IP.
```

Comment:
```suggestion
  * **Section 1:** Ensure that the telemetry data is sent to the collector. This sets the environment variable `OTEL_EXPORTER_OTLP_ENDPOINT` with the host IP. It does this by calling the downward API to pull the host IP.
```

Reply:
@zuluecho9, this looks like a duplication of a previous comment, so I'm just going to close it.